README.md

```diff
@@ -75,6 +75,72 @@ python manage.py test
 flake8 .
 ```
 
+### Windows 10 Users
+ Please install and set up the following packages first, upgrade if you find the package is already installed:
+
+* Download and install [Python3](https://www.python.org/downloads/) and ensure to 'Add path' while installing. Run the 'python3 -V' command to see the version you have installed.
```

Comment:
The only way one can click 'add path' is to install the python package as administrator, right? If this is the case, please add that to the instructions.

Reply:
After adding PATH in the administrator, then the VS Code will be installed as is

Reply:
VS Code is a text editor, code editor. It's just for writing code, anyone could use any code editor of their choice, vscode isn't the only existing one.

Reply:
Done.